tests/unit_tests/test_game_resolution.py

```
@@ -37,68 +37,39 @@
-        resolve_game(team_outcomes)
-
-    team_outcomes = [{ArmyOutcome.CONFLICTING}, {ArmyOutcome.CONFLICTING}]
-    with pytest.raises(GameResolutionError):
-        resolve_game(team_outcomes)
+    ERROR = 0
+    DRAW = 1
+    WIN = 2
+    LOSS = 3
+    #        Victory  Defeat   Recall   Draw     Unknown  Conflicting
+    grid = [[ERROR,   WIN,     WIN,     WIN,     WIN,     WIN  ]  # Victory
+            [LOSS,    DRAW,    DRAW,    ERROR,   ERROR,   ERROR]  # Defeat
```

Ruff (0.14.3)

Indexed access to type `list` uses type `tuple` instead of an integer or slice

(RUF016)